express-delete/server.js

```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const app = express();
+app.get('/api/grades', (req, res) => {
+  const gradesArray = [];
+  for (const grade in grades) {
+    gradesArray.push(grades[grade]);
+  }
+  res.json(gradesArray);
+});
+
+app.delete('/api/grades/:id', (req, res) => {
+  console.log(req.params.id);
+  delete grades[req.params.id];
```

Comment:
We should have some sort of check that the `id` passed in already exists on the object. If so, then we want to delete it. If not, we can respond with a message or status of the id not being found.